keyboards/jj_ergo/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "jj_ergo",
+    "url": "",
+    "maintainer": "Neil Kettle",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k10", "x":0, "y":1, "w":1.5},
+                {"label":"k12", "x":1.5, "y":1, "w":1.5}
+            ]
+        }
```

Comment:
This hasn't been changed from the template - you can use keyboard-layout-editor and https://qmk.fm/converter to get an info.json for Configurator support.

Reply:
Easier way:

Use the contents of the `LAYOUT_ergodox` tree from [layouts/default/ergodox/info.json](https://github.com/qmk/qmk_firmware/blob/91c8a9314aed50d4220ce18e7f3f1e0476e44286/layouts/default/ergodox/info.json).